### DISCUSSION TITLE
I can't access individual event pages; I always end up on the event overview.

### DISCUSSION THREAD

**Discussion by ebussinetz** [liedekef/events-made-easy#626] [Q&A] [answered]
Now almost everything works perfectly, but I can't access individual event pages; I always end up on the event overview.
There is probably a simple solution for this too, but despite many attempts I haven't figured it out yet...
And once again, thank you very much!

https://www.wmwebservice.de/divitest/termine/

**Comment by liedekef** [comment]
Did you read https://www.e-dynamics.be/wordpress/installation-and-upgrade/ ? Especially the part about the special events page that is used by EME? I'm guessing that's your problem here: what is your EME setting for that special events page? See the EME settings, tab Events, setting "Events page"

**Comment by ebussinetz** [comment]
OK, I have now set "Events" as eventspage and "events" as permalink (should I do this so?). 
Now if I klick on the link of an event, there comes a page called "Events" but it is empty.
In the documentation, you have sent to me, is written, what we should not do, but not if we had done it, waht to do then to repair ...

My setiing in Settings/Events for a single evants page is:
#_STARTDATE - #_STARTTIME<br />#_TOWN<br />#_NOTES<br />#_ADDBOOKINGFORM<br />#_MAP

A problem is, when I set a different side than "Termine" where my overview is, then the single events pages are no subpage from "Termine", an dI would like to set the parent-menubutton to an active color.

I compared the settings with my other website, spiritualchor.de, where everything works, and found no relevant differences.

**Comment by liedekef** [comment]
The content for a single event is rendered in place of the special Events page. It shouldn't matter wether or not the special events page is a subpage of Termine, but Termine is just German for Events, so you're creating quite a mess there :-)
Since the content is empty, I'm assuming something might be off with your WP permalink settings ... compare those in the 2 wp installations too. Also, first try with the default 24-template from WP, it might also be a template thing (EME uses the WP filter "the_content" to replace the content of the special events page with the content of an event, but your theme needs to be WP compliant for this to work correctly). Also, since you have permalinks activated: set this active in EME too.

**Comment by ebussinetz** [comment]
OK, thank you!!!
It has to do with the DIVI-Theme, with a standard theme it runs.
I have to ask them ...

**Reply by liedekef** [reply]
hum ... is that a free theme? Can I install it to test?

**Comment by ebussinetz** [comment]
It is not free but I have got a lifetime-licence, you can download it here, it should work but you can´t update it:
(link deleted)

I´m wondering, that the shortcode of the list works but not for a single event.

**Reply by liedekef** [reply, marked as answer]
I tried your theme and it works as expected here ...

**Comment by ebussinetz** [comment]
I set up a template in the DIVI theme builder that doesn't modify the body. It works now. Unfortunately, it doesn't look very nice.
Have you ever made the event pages a popup, similar to this one, when you click on the individual tiles?
https://www.wmwebservice.de/divitest/verzeichnis/vereine-institutionen/

**Reply by liedekef** [reply]
I don't know divi and I'm not much of a theme/layout guy :-) EME is set up in such a way that everything fits in the style of your theme, but the theme needs to do the work :-) In this case, if you know how to create the popups html-wise, you can change the event listing so it does that. But don't ask me how in divi, maybe you can ask them for the html-part needed so a click on a link results in a popup (by default EME just creates a ul-style list but is freely adaptable via templates for the header/entry/footer)

**Reply by liedekef** [reply]
btw, divi popup: https://www.divithemeexamples.com/how-to-add-popups-to-divi/

**Reply by ebussinetz** [reply]
Yes I know. I would find it out, for now is enough, is good enough and with your plugin, the website is more comfortable as it was with only a table with the events.

I have one more small question: Is it possible to group the dates by month?
e.g.

August
- Date 1
- Date 2

October
- Date 3
- Date 4

**Reply by liedekef** [reply]
See the doc at https://www.e-dynamics.be/wordpress/eme_events/ : the parameter showperiod=monthly is what you want. Example: [eme_events showperiod=monthly]. The date format for the monthly showperiod can be changed in the Settings page

**Reply by ebussinetz** [reply]
Thank you again! Your support is the best I ever had! Have a nice weekend!
